src/Compilers/CSharp/Portable/Errors/.editorconfig

```diff
@@ -0,0 +1,4 @@
+# CSharp code style settings:
```

Comment:
😕 Why are we adding a new file here?

Reply:
We want to run the analyzer specifically on the enums in this file. We're not interested in having the warning on the rest of the enums in the compiler at this time.